CONTRIBUTING.rst

```diff
@@ -97,15 +97,18 @@ Ready to contribute? Here's how to set up `orsopy` for local development.
 Pull Request Guidelines
 -----------------------
 
-Before you submit a pull request, check that it meets these guidelines:
+Before you submit a pull request of your feature branch against the `main` branch of the `orsopy` repository,
+check that it meets these guidelines:
 
-1. The pull request should include tests.
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
    feature to the list in README.rst.
-3. The pull request should work for Python 3.5, 3.6, 3.7 and 3.8, and for PyPy. Check
-   https://travis-ci.com/reflectivity/orsopy/pull_requests
-   and make sure that the tests pass for all supported Python versions.
+1. The pull request should include tests for the new functionality. Run the tests in your local machine with `pytest`.
+3. The pull request should work for Python 3.5, 3.6, 3.7 and 3.8, and for PyPy. To make sure that the tests pass
```

Comment:
```suggestion
3. The pull request should work for Python 3.6, 3.7, 3.8, 3.9, 3.10, 3.11 and for PyPy. To make sure that the tests pass
```

3.6 and 3.7 are kind of supported, in that there's a desire for older Python versions to keep working if possible. 3.5 is too old. So `[3.6, 3.7]` if we can keep it working, but definitely `[3.8, 3.9, "3.10", "3.11", "3.12"]`.